nemo/backends/pytorch/common/losses.py

```diff
@@ -34,24 +34,8 @@ class SequenceLoss(LossNM):
     @property
     def input_ports(self):
         """Returns definitions of module input ports.
-
-        log_probs:
-            0: AxisType(BatchTag)
-
-            1: AxisType(TimeTag)
-
-            2: AxisType(ChannelTag)
-
-        targets:
-            0: AxisType(BatchTag)
-
-            1: AxisType(TimeTag)
-
         """
-        return {
-            'log_probs': NeuralType({0: AxisType(BatchTag), 1: AxisType(TimeTag), 2: AxisType(ChannelTag),}),
-            'targets': NeuralType({0: AxisType(BatchTag), 1: AxisType(TimeTag)}),
-        }
```

Comment:
Moreover, this description seems to be perfectly fitting the actually returned definitions of ports...